plugins/janus_videoroom.c

```diff
@@ -7489,14 +7503,41 @@ static void janus_videoroom_recorder_create(janus_videoroom_publisher *participa
 		participant->drc = rc;
 	}
 }
+// CARBYNE-S3-UPLOAD
+
+int create_recording_json(char *filename, char *codec)
+{
+	char *jsonObj;
+	asprintf(&jsonObj, "{\n \"recordingName\" : \"%s\" ,\n \"directory\" : \"/home/ubuntu/recordings/\" ,\n\"codec\" : \"%s\"\n}", filename, codec);
+	char *fullFileName;
+	// removeSubstr(filename,".mjr");
+	asprintf(&fullFileName, "/home/ubuntu/RecJSON/%s.json", filename);
+	printf("[file debug]------%s\n", fullFileName);
```

Comment:
Does printf go to wgw logs? cloud watch?

Reply:
yes